prime-router/settings/organizations.yml

```diff
@@ -106,6 +106,11 @@
       customerStatus: active
       schemaName: classpath:/metadata/fhir_transforms/senders/original-pipeline-transforms.yml
       format: HL7
+    - name: ignore-mars-otc
```

Comment:
A little unrelated, but I realized we should have a mars otc sender for test purposes